File: examples-sdk/java/501-villog-stdout-dev/Main.java
// 501-villog-stdout-dev — Java SDK equivalent
// Compile: vil compile --from java --input 501-villog-stdout-dev/Main.java --release

import dev.vil.*;

public class Main {
    public static void main(String[] args) {
        VilServer server = new VilServer("app", 8080);
        server.compile();
    }
}

File: examples-sdk/csharp/501-villog-stdout-dev/Main.cs
// 501-villog-stdout-dev — C# SDK equivalent
// Compile: vil compile --from csharp --input 501-villog-stdout-dev/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("app", 8080);
server.Compile();
